build.sbt

```diff
@@ -63,7 +63,7 @@ val testingDependencies = Seq(
   "com.typesafe.akka" %% "akka-testkit" % "2.5.3" % "test",
   "org.scalactic" %% "scalactic" % "3.0.3" % "test",
   "org.scalatest" %% "scalatest" % "3.0.3" % "test",
-  "org.scalacheck" %% "scalacheck" % "1.13.+" % "test",
+  "org.scalacheck" %% "scalacheck" % "1.13.+",
```

Comment:
Why "test" scope removed?

Reply:
Cause to inherit this class from other module we need him to be not in tests. So that's why we need generators in main source.